### PR TITLE
Use a second set of images/icons.

### PR DESCRIPTION
**Pull request by sumaiyamannan**
Hi,

This creates the feature of using a second set of images or icons to the plugin.

Regards,

Sumaiya

### CODE REVIEW

File: activitystatusimg.php [hunk 1]
```diff
@@ -1,61 +1,57 @@
 <?php
-
 define('NO_DEBUG_DISPLAY', true);
-
 require(dirname(__FILE__).'/../../config.php');
 require_once($CFG->dirroot.'/lib/completionlib.php');
 
 $idnumber = required_param('idnumber', PARAM_TEXT);
+$iconset = optional_param('iconset', '', PARAM_TEXT);
```
Comment (danmarsden) [inline activitystatusimg.php]: does this need to be param_text? - can it be PARAM_ALPHA or PARAM_INT?

File: activitystatusimg.php [hunk 2]
```diff
@@ -1,61 +1,57 @@
...
 
-print_icon('incompleteacticon');
+print_icon($incompleteacticon);
 return;
 
-
-
-
 function print_unknownactivity() {
+    global $CFG;
```
Comment (danmarsden) [inline activitystatusimg.php:50]: accidental change adding global $CFG?

Comment (danmarsden) [reply activitystatusimg.php:50]: nope - bug fix, closing :-)

File: activitystatusimg.php (hunk 1 above)
Comment (sumaiyamannan) [reply activitystatusimg.php]: done

**Pull request merged**